src/provider/camunda-platform/properties/FormField.js

```diff
@@ -361,7 +361,7 @@ function ValueList(props) {
 
```

Comment:
Please add `Closes #589` to the commit message.

This establishes the relevant context needed to track back the fix to the bug.